testing/cc/non_module_deps.bzl

```diff
@@ -0,0 +1,11 @@
+load(
+    "//tests:nixpkgs_repositories.bzl",
+    "nixpkgs_repositories",
+)
+
+def _non_module_deps_impl(ctx):
+    nixpkgs_repositories(bzlmod = True)
+
+non_module_deps = module_extension(
+    implementation = _non_module_deps_impl,
+)
```

Comment:
It could be I haven't had enough coffee, but I can't figure out why `testing/cc/non_module_deps.bzl` is showing as changed here :sweat_smile: 

I know this PR depended on the #325 one, but that was merged. Locally I see no difference between these two branches for this file:
```
git diff origin/bzlmod-go origin/master -- testing/cc/non_module_deps.bzl
```

Reply:
Hmm, I think GitHub is diffing against the merge base instead of master, i.e.
```
git diff --stat $(git merge-base origin/master origin/bzlmod-go) origin/bzlmod-go
```
instead of
```
git diff --stat origin/master origin/bzlmod-go
```

Reply:
I think it's a `git merge-base` thing.